Solver.hpp

```diff
@@ -28,7 +28,7 @@ namespace Connect4 {
 
 class Solver {
  private:
-  static const int TABLE_SIZE = 24; // store 2^TABLE_SIZE elements in the transpositiontbale
+  static constexpr int TABLE_SIZE = 24; // store 2^TABLE_SIZE elements in the transpositiontbale
   TranspositionTable < uint_t < Position::WIDTH*(Position::HEIGHT + 1) - TABLE_SIZE >, Position::position_t, uint8_t, TABLE_SIZE > transTable;
```

Comment:
added, to provide consistency across the code.

note: according to https://stackoverflow.com/questions/41125651/constexpr-vs-static-const-which-one-to-prefer
there is no difference between static const and constexpr for scalar type